moulinette/interfaces/api.py

```diff
@@ -80,6 +80,24 @@ def set_defaults(self, **kwargs):
     def get_default(self, dest):
         return self._parser.get_default(dest)
 
+    def add_arguments(self, arguments, extraparser, format_arg_names=None, validate_extra=True):
```

Comment:
Yes, very possibly, but right now I was having a very "make this only function readable" approach, a lot of things should be simplified in moulinette but I didn't want to put too much stuff in one PR.